roxie/ccd/ccdfile.cpp

```diff
@@ -1128,11 +1128,20 @@ class CRoxieFileCache : implements IRoxieFileCache, implements ICopyFileProgress
         {
             CriticalBlock b(crit);
             Linked<ILazyFileIO> f = files.getValue(localLocation);
-            if (f && f->isAlive())
+            while (f && f->isAlive())
```

Comment:
I may be misreading, but I think the only place it will try again is if (!f), so the loop will abort straight away